CHANGELOG.md

```diff
@@ -37,7 +37,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
-None
+### Features 
+
+  * Add CONTROLLER, and VALIDATOR PartyTypes for contract execution. 
+
+### Improvements 
```

Comment:
If there's something else that needs to be changed, maybe also remove this `### Improvements` line. It's not hurting anything, though, and we'll most likely need it in the long run, so no need to remove it unless there's other changes to push up too.